.pre-commit-config.yaml

```diff
@@ -15,6 +15,11 @@ repos:
 #    hooks:
 #      - id: mypy
 #        entry: mypy
+  - repo: http://github.com/ambv/black
+    rev: 24.4.2
+    hooks:
+      - id: black
+        language_version: python3.10
```

Comment:
I think
```yaml
  - repo: http://github.com/pycqa/isort
```
is missing, no?

Reply:
No this is not missing

Reply:
Not sure how I missed it but yep, it's there.